lib/IPv4.js

```diff
@@ -383,6 +383,23 @@ class IPv4 {
     return ipAddresses;
   }
 
+  /**
+   * Get the number of ip addresses between two given ip addresses
+   * @param {string} ip1
+   * @param {string} ip2
+   * @returns {number}
+   */
+  getAdressesDistance(ip1, ip2) {
+    if (!this.isValid(ip1) || !this.isValid(ip2)) {
+      throw new Error('Invalid ip address!');
+    }
+    const intIp1 = this.toInteger(ip1);
+    const intIp2 = this.toInteger(ip2);
+		
+		const distance = Math.abs(intIp1 - intIp2);
```

Comment:
Local variable `distance` is redundant. You can return it without declaring a variable.

Reply:
thx, fixed